app/assets/javascripts/admin/tagify.js

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
@garytaylor Courtfinder started in 2015 and coffeescript was still the default option in Rails community.

I prefer JS over coffeescript and I would be happy to discuss about moving away from it.